packages/push-notifications/src/apns/PushNotificationsApnsApi.ts

```diff
@@ -53,14 +53,15 @@ export class PushNotificationsApnsApi {
    * Response for `push-notifications-apns/get-device-info`
    *
    * @param connectionId The connection ID string
+   * @param threadId get-device-info message ID
    * @param deviceInfo The APNS device info
    * @returns Promise<void>
    */
-  public async deviceInfo(connectionId: string, deviceInfo: ApnsDeviceInfo) {
+  public async deviceInfo(connectionId: string, threadId: string, deviceInfo: ApnsDeviceInfo) {
```

Comment:
This is getting to a point I think where an object as input would be better. Connection and threadid are both uuids so it might become annoying to deal with later.